tests/test_pytest/test_plugin_schema/bad/product_defaults/bases/colormap.yaml

```diff
@@ -0,0 +1,78 @@
+---
+error: ValidationError
+error_pattern: ".*123 is not of type 'string'.*"
+plugin:
+  name: 123
```

Comment:
@jsolbrig So we ended on module-based plugins requiring valid Python identifiers as "name", and YAML-based plugins do NOT require valid Python identifiers?  I think that is fair.

Reply:
For now, yes. I'd prefer, though, if we could reinstate the requirement for YAML eventually. It would make some things possible that currently aren't.